docs/dev/api_integration.md

```diff
@@ -5,161 +5,55 @@ type: documentation
 - [Traces][6]
 - [Incidents][7]
-- [Security Events][8]
-
-This page provides instructions for creating an API integration in the `integrations-extras` repository. For more information about why you would want to create an API-based integration, see [Creating your own solution][9]. Since API integrations do not use the Datadog Agent to collect data, you need to create an informational tile-only listing once your development work is complete. 
-
-## Setup
-
-### Prerequisites
-
-- You must have an [API key][10] and [application key][11].
-- Determine which [Datadog site][12] you want to use.
-
-An API key is required to submit data to a Datadog API endpoint. An application key is required to query data from Datadog or to create resources within the Datadog site. For more information, see [API and Application Key][13].
-
-Create a connection to Datadog in your company's platform using the API key, application key, and site URL. 
-
-### Create an OAuth client
-Instead of requesting these credentials directly from a user, Datadog recommends using an [OAuth client][14] to handle authorization and access for API-based integrations. For more information, see [OAuth for Integrations][15] and [Authorization Endpoints][16]
-
-You can explore examples of existing API integrations in the `integrations-extras` repository such as [Vantage][17].
-
-## Set up a directory and fork the `integrations-extras` repository
-
-
-1. Create a `dd` directory:
-
-   {{< code-block lang="shell" >}}mkdir $HOME/dd{{< /code-block >}}
-   
-   The Datadog Development Toolkit expects you to be working in the `$HOME/dd/` directory. This is not mandatory, but working in a different directory requires additional configuration steps.
-
-2. Clone the `integrations-extras` repository:
-
-   {{< code-block lang="shell" >}}git clone git@github.com:DataDog/integrations-extras.git{{< /code-block >}}
-
-## Install and configure the Datadog development toolkit
-
-The Agent Integration Developer Tool allows you to create scaffolding when you are developing an integration by generating a skeleton of your integration tile's assets and metadata. For instructions on installing the tool, see [Install the Datadog Agent Integration Developer Tool][23].
 
-After you install the Developer tool, configure it for the `integrations-extras` repo:
+You can also include out-of-the-box assets like Monitors, Dashboards, and Log Pipelines. When a user clicks "Install" on your integration tile, they will be guided to your setup instructions, and all out-of-the-box dashboards and recommended monitors packaged with your integration will appear in the user's account.
 
-Set `integrations-extras` as the default working repository:
+This page provides instructions for creating an API integration. Since API integrations do not use the Datadog Agent to collect data, you need to create an informational tile-only listing to display your offering on the Integrations page once your development work is complete.
```

Comment:
I think this should be moved up with the sentence: 
 This method is ideal for Technology Partners that are SaaS based, and have an existing platform that authenticates users.